wpe/src/main/cpp/browser/pageeventobserver.cpp

```diff
@@ -28,7 +30,9 @@ constexpr JavaMethodDesc javaMethods[PageEventObserver::NB_JAVA_METHODS] = {
         { "onUriChanged",            "(Ljava/lang/String;)V" },
         { "onTitleChanged",          "(Ljava/lang/String;ZZ)V" },
         { "onInputMethodContextIn",  "()V" },
-        { "onInputMethodContextOut", "()V" }
+        { "onInputMethodContextOut", "()V" },
+        { "enterFullscreenMode",       "()V" },
+        { "exitFullscreenMode",        "()V" }
```

Comment:
Personally I'm not a big fan of trailing comas because you don't know if it is the truly end of the list or a git merge that has deleted one entry (already happened...)

Reply:
I don't have a strong preferenmce, to be honest... as long as we pick one style and try to keep it consistent, I'm happy 😄

Reply:
fixed